src/boards/include/boards/pimoroni_tiny2040.h

```diff
@@ -14,6 +14,7 @@
 
 // For board detection
 #define PIMORONI_TINY2040
+#define PIMORONI_TINY2040_8MB
```

Comment:
In a similar vein, _perhaps_ it's worth adding `#define PIMORONI_PICOLIPO` to `pimoroni_picolipo_4mb.h` and `pimoroni_picolipo_16mb.h` ? :shrug: 

Hmmm, and (merely for the sake of "symmetry") I wonder if it might be worth adding a `pimoroni_tiny2040_8mb.h` which just `#include`s `pimoroni_tiny2040.h` ?

Reply:
I'm in favour of adding `#define PIMORONI_PICOLIPO` but thought it best to do that as a separate PR from this?

As for the second suggestion. I would have but as the 8MB product is already out there I did not want to break compatibility for anyone.